config/localization/en/app.json

```diff
@@ -1340,7 +1340,6 @@
         "LAUNCH_INCENTIVES_DESCRIPTION":"Each season, you can earn rewards for trading on dYdX Chain. Trade to earn your spot on the leaderboard.",
         "LEADERBOARD": "Leaderboard",
         "POINTS":"{POINTS} points",
-        "POWERED_BY":"POWERED BY",
```

Comment:
duplicate key (it's already in `app.general.powered_by` so this isn't even recognized in the generated `app.ts` file)